server/test/auth.test.js

```diff
@@ -0,0 +1,19 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const { app } = require('../app.js');
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Mocha test for auth controller', () => {
+  const message = 'You have successfully logged out';
+
+  describe('GET /auth/logout ', () => {
+    it('should return a successful response', async () => {
+      const res = await chai.request(app).get('/auth/logout');
```

Comment:
ust so we are aware, GET isn't the verb that makes the most sense for a /logout path! (see this [article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!